### PR TITLE
Complete desktop version

### PR DESCRIPTION
**Pull request by PromzzyKoncepts**
# Hello there 😁 👍 
<h3>I trust you are having a great day</h3>

Below are the new updates to this repository:
- I built the desktop version using media queries
- Checked for responsiveness
- Corrected linter errors
- Committed and push to this repo

You can view the <strong>live demo</strong> [here](https://promzzykoncepts.github.io/My-Portfolio/)
<h3>Thank you! 🤟 </h3>
<h4>Happy Reviewing! <br>Hoping to get approved</h4>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,21 +20,21 @@
...
         </header>
         <!-- header  section -->
         <section class="first-section">
           <div class="header-intro">
             <h1 class="header-text">
-              Hey There.
+              Hey There. <br>
               I’m Promise
             </h1>
             <h2 class="header-smaller-text"> I am a Software Developer </h2>
```
Comment (Reem-lab) [inline index.html:40]: - [ ] Great job so far just a small update here please align the header in the center to match the Figma design🙏🏻✅

- Your design 
image

- Figma design
image

File: index.html [hunk 2]
```diff
@@ -274,27 +383,27 @@ <h2>
...
+              
+              <div>
+                <button type="submit" id="contact-button" class="project-btn1">Get in touch</button>
+              </div>
+            <!-- <hr class="hr"> </hr> -->
+          </div>
+          <div><input type="text" name="name" id="full-name" placeholder="Last name" class="fullname" maxlength="30" required><hr></div>
+          
         </section>
       </form>
```
Comment (Reem-lab) [inline index.html:409]: - [ ] Kindly align the social links to match the Figma design as much as possible 🙏🏻🙏🏻✅✅
- Figma design
image

- Your design
image

**Review by Reem-lab — CHANGES_REQUESTED** [review]
## Hi @PromzzyKoncepts, 👋🏻👋🏻 

Good job so far!🔥🔥🔥✨✨
There are some issues that you still need to work on to go to the next project but you are almost there!
Personally I appreciate you great job but just small things ti improve to nail it ✅🔥🔥🚀🚀 
keep it up🚀


## Highlights✨
- No linters error .✅✅
- Descriptive pull request ✅✅
- Good commits messages✅✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -172,7 +217,27 @@ <h3>Professional Art Printing Data</h3>
                 </div>
                 <button type="button" class="project-btn2">See Project</button>
```
Comment (Reem-lab) [inline index.html:218]: - [ ] Kindly Fix the font weight to the text for the button to match the Figma design it is important for user experience.
 and add styling: Active, Hover, and Pressed please follow the styling and add them to your button to match the Figma

**Review by Reem-lab — CHANGES_REQUESTED** [review]

**Review by fobadara — CHANGES_REQUESTED** [review]
# REQUESTED CHANGES ♻️ 

Hi @PromzzyKoncepts, 

Good job so far! 👍🏽 👍🏽 
There are some issues that you still need to work on to go to the next project but you are almost there! 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fobadara** [comment]
- [ ] Good job attempting to implement the suggestion in [this comment](https://github.com/PromzzyKoncepts/My-Portfolio/pull/8#discussion_r915388707) 👍🏽  but you are not quite done. You still need to move the icons a little to the right so that your design can match the Figma design. Take note of the red vertical lines in the images below:

|   Your Design                      |   Figma Design                  |
| ------------------------ | ------------------------ |
| Promise_ Portfolio_footer | Microverse-Student-Project-1 – Figma_footer |


[OPTIONAL] There are also two arrows pointing to a faint line in the Figma design above. Please include it. One of the goals of giving you template is to train your eyes for details.

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: 

Hi @PromzzyKoncepts, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors :heavy_check_mark: 
- [ ] Good commit history :heavy_check_mark: 
- [ ] Descriptive Pull Request :heavy_check_mark: 
- [ ] Good implementation of all the changes required :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**